pmd-ui/src/main/java/net/sourceforge/pmd/util/fxdesigner/EventLogController.java

```diff
@@ -35,6 +37,8 @@
 
     private final DesignerRoot designerRoot;
 
+    private final Duration timeDuration = Duration.ofMillis(100);
```

Comment:
This should be static final, and have a more descriptive name. Good practice is asking yourself, what is this object used for? A good name should be documentation in itself.